src/components/LineItem.jsx

```diff
@@ -12,144 +12,160 @@ import {
+
   const handleEventChange = (event, prop) => {
     updateLineItem(mapID, prop, event.target.value);
   };
 
+  const handleLineItemCollapse = () => {
+    setOpen(!open);
+  }
+
   return (
-    <Grid container spacing={2} mb={4}>
-      <Grid item md={4}>
-        <TextField
-          fullWidth
-          required
-          error={lineItemError?.item}
-          label="Item"
-          value={lineItem.item}
-          onChange={(event) => handleEventChange(event, 'item')}
-        />
-      </Grid>
-      <Grid item md={4}>
-        <TextField
-          fullWidth
-          label="Description"
-          value={lineItem.description}
-          onChange={(event) => handleEventChange(event, 'description')}
-        />
-      </Grid>
-      <Grid item md={4}>
-        <TextField
-          fullWidth
-          required
-          error={lineItemError?.unitOfMeasure}
-          label="Unit of Measure"
-          value={lineItem.unitOfMeasure}
-          onChange={(event) => handleEventChange(event, 'unitOfMeasure')}
-        />
-      </Grid>
-      <Grid item md={4}>
-        <TextField
-          fullWidth
-          required
-          error={lineItemError?.orderedQuantity}
-          label="Ordered quantity"
-          value={lineItem.orderedQuantity}
-          onChange={(event) => handleEventChange(event, 'orderedQuantity')}
-        />
-      </Grid>
-      <Grid item md={4}>
-        <TextField
-          fullWidth
-          required
-          error={lineItemError?.acknowledgedQuantity}
-          label="Acknowledged quantity"
-          value={lineItem.acknowledgedQuantity}
-          onChange={(event) => handleEventChange(event, 'acknowledgedQuantity')}
-        />
-      </Grid>
-      <Grid item md={4}>
-        <TextField
-          fullWidth
-          required
-          error={lineItemError?.price}
-          label="Price"
-          value={lineItem.price}
-          onChange={(event) => handleEventChange(event, 'price')}
-          InputProps={{
-            endAdornment: (
-              <InputAdornment position="end">
-                <Tooltip title="must be a number" arrow>
-                  <InfoOutlined />
-                </Tooltip>
-              </InputAdornment>
-            )
-          }}
-        />
-      </Grid>
-      <Grid item xs={4}>
-        <FormControl fullWidth>
-          <InputLabel>Acknowledgement Status</InputLabel>
-          <Select
-            label="Acknowledgement Status"
-            value={lineItem.acknowledgementStatus}
-            onChange={(event) =>
-              handleEventChange(event, 'acknowledgementStatus')
-            }
-          >
-            {poLineStatusCodes.map((poLineStatusCode) => (
-              <MenuItem
-                value={poLineStatusCode.code}
-                key={poLineStatusCode.code}
+    <Box>
+      <Button variant='text' onClick={handleLineItemCollapse} endIcon={open ? <KeyboardArrowDown /> : <KeyboardArrowRight />}>{`Line Item: ${index}`}</Button>
```

Comment:
I like this addition a lot, since it is collapsible how does it look if we have it also have the item's name somewhere

for example
0: MYITEM
Line Number: 0 (MYITEM)

Reply:
With the names in the collapse menu the arrows don't line up anymore but otherwise it works great!
![image](https://user-images.githubusercontent.com/94586121/216692587-867f31d1-fd6a-43a5-8894-2f97f4773bb6.png)

Reply:
How does it look when changing it to startIcon so its on the left?